### PR TITLE
Fixed #107 #108

### LINKED ISSUE #107: Add pep/pop sort function to annotation

### PR DESCRIPTION
**Pull request by khoroshevskyi**
1) Added get annotation by providing list of registry paths
2) Added `pep_type` sort function to annotations (pep/pop)

### CODE REVIEW

File: pepdbagent/modules/annotation.py [hunk 1]
```diff
@@ -77,6 +78,7 @@ def get(
             [Default: filter won't be used]
         :param filter_start_date: Filter start date. Format: "YYYY/MM/DD"
         :param filter_end_date: Filter end date. Format: "YYYY/MM/DD". if None: present date will be used
+        :param pep_type: Get pep with specified type. Options: ["pep", "pop"]. Default: None, get all peps
```
Comment (nleroy917) [inline pepdbagent/modules/annotation.py:81]: 👍🏼

File: pepdbagent/modules/annotation.py [hunk 2]
```diff
@@ -94,6 +96,10 @@ def get(
                 offset=0,
                 results=found_annotation,
             )
+
+        if pep_type not in [None, "pep", "pop"]:
+            raise ValueError(f"pep_type should be one of ['pep', 'pop'], got {pep_type}")
```
Comment (nleroy917) [inline pepdbagent/modules/annotation.py:101]: 👍🏼

File: pepdbagent/modules/annotation.py [hunk 3]
```diff
@@ -463,3 +481,82 @@ def get_project_number_in_namespace(
...
+                            Projects.tag == tag,
+                            or_(
+                                Projects.namespace.in_(admin_tuple),
+                                Projects.private.is_(False),
+                            ),
+                        )
+                    )
+                except RegistryPathError as err:
+                    _LOGGER.error(str(err), registry_paths)
+                    continue
```
Comment (nleroy917) [inline pepdbagent/modules/annotation.py:521]: Is it a good idea to ignore registry paths that are invalid? Trying to think of a scenario where this bites us.

**Review by nleroy917 — APPROVED** [review]
I think that this looks good, some things I made a comment on about masking errors, otherwise 👍🏼

File: pepdbagent/modules/annotation.py (hunk 3 above)
Comment (khoroshevskyi) [reply pepdbagent/modules/annotation.py:521]: Solved this issue by returning None instead of skipping error

**Pull request merged**